connect/transforms/src/main/java/org/apache/kafka/connect/transforms/util/Requirements.java

```diff
@@ -40,13 +40,27 @@ public static void requireSchema(Schema schema, String purpose) {
         return (Map<String, Object>) value;
     }
 
+    public static Map<String, Object> requireMapOrNull(Object value, String purpose) {
+        if (value == null) {
```

Comment:
If I remember correctly you don't like one-liners of that sort too much, but otherwise it's an option, since you use it when you pass the argument above: 
`return value == null ? null : requireMap(value, purpose);`